apps/wallet/src/routes/Deposit/Buy.tsx

```diff
@@ -55,7 +71,13 @@ export const Buy = () => {
           setShowDoneButton(true);
           break;
         default:
-          return;
+          break;
+      }
+
+      // Hook into TRANSAK_WIDGET_CLOSE event to navigate back to Wallet home page upon user successfully purchasing, then tapping "Back to App"
```

Comment:
Best way I could come up with to navigate a user back home when clicking this button